packages/lib-content/src/translations/i18n.js

```diff
@@ -17,6 +18,7 @@ libI18n.use(initReactI18next).init({
 
 libI18n.addResourceBundle('en', 'translation', require('./en.json'))
 
+/* In FEM there's an i18n instance for each library, and each instance has its own functions.*/
```

Comment:
See https://github.com/zooniverse/front-end-monorepo/pull/3900#issuecomment-1320871722 for more details.